### ISSUE TITLE
missing boolean value

### ISSUE THREAD

**Issue by soulbrotherfunky** [ebaauw/homebridge-hue#1198]
#### Issue

Hi, just ran into some problems with 14.4., maybe it has to do with the homebridge 2.0 beta? going back to 14.3 solves the issue for now.
#### Log Messages

[11.11.2024, 13:18:48] [homebridge-hue] TypeError: stealth: missing boolean value
    at newTypeError (file:///var/lib/homebridge/node_modules/homebridge-hue/node_modules/hb-lib-tools/lib/OptionParser.js:31:52)
    at Function.toBool (file:///var/lib/homebridge/node_modules/homebridge-hue/node_modules/hb-lib-tools/lib/OptionParser.js:98:13)
    at Object._callbacks.<computed> [as stealth] (file:///var/lib/homebridge/node_modules/homebridge-hue/node_modules/hb-lib-tools/lib/OptionParser.js:604:40)
    at OptionParser.parse (file:///var/lib/homebridge/node_modules/homebridge-hue/node_modules/hb-lib-tools/lib/OptionParser.js:876:29)
    at HueDiscovery.discover (file:///var/lib/homebridge/node_modules/homebridge-hue/node_modules/hb-hue-tools/lib/HueDiscovery.js:142:8)
    at HuePlatform.findBridges (file:///var/lib/homebridge/node_modules/homebridge-hue/lib/HuePlatform.js:424:43)
    at HuePlatform.accessories (file:///var/lib/homebridge/node_modules/homebridge-hue/lib/HuePlatform.js:202:39)
    at file:///var/lib/homebridge/node_modules/homebridge/src/bridgeService.ts:495:24
    at new Promise (<anonymous>)
    at BridgeService.loadPlatformAccessories (file:///var/lib/homebridge/node_modules/homebridge/src/bridgeService.ts:489:12)```
```

#### Debug Files
<!-- Please attach homebridge-hue.json.gz, see https://github.com/ebaauw/homebridge-hue#debug-dump-file -->

<!-- Please attach homebridge.log.gz here, see https://github.com/ebaauw/homebridge-hue#debug-log-file -->

**Comment by ebaauw** [comment]
I think there's a problem with your config.json.  What's the value for the `stealth` key?

**Comment by ebaauw** [comment]
Should be fixed in v0.14.5.

**Comment by soulbrotherfunky** [comment]
thanks! 👍
